sdk/src/client/types.ts

```diff
@@ -39,20 +39,31 @@ export interface Channel {
     nonce: bigint; // Unique per channel with same participants and adjudicator
 }
 
+/**
+ * Channel status enum
+ */
+export enum StateIntent {
+    OPERATE, // Operate the state application
+    INITIALIZE, // Initial funding state
+    RESIZE, // Resize state
+    FINALIZE, // Final closing state
+}
+
 /**
  * Channel state structure
  */
 export interface State {
-    data: Hex; // Application data encoded, decoded by the adjudicator for business logic
+    intent: StateIntent; // Intent of the state (e.g., INITIAL, ACTIVE, FINAL)
     version: bigint; // Version of the state, incremented for each update
+    data: Hex; // Application data encoded, decoded by the adjudicator for business logic
     allocations: [Allocation, Allocation]; // Combined asset allocation and destination for each participant
     sigs: Signature[]; // stateHash signatures
 }
 
 /**
  * Adjudicator status enum
  */
-export enum AdjudicatorStatus {
+export enum adjudicatorStatus {
```

Comment:
If this references `ChannelStatus` Solidity enum, then it can be safely removed from SDK.